.github/workflows/ci.yml

```diff
@@ -11,8 +11,10 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-18.04, macos-latest, windows-latest]
     steps:
+    - name: List GLIBC versions
+      run: nm -D /lib/x86_64-linux-gnu/libc.so.6 | grep " GLIBC_2" | sort -V
```

Comment:
This doesn't run on platform other than linux and causing CI fail on macos, windows now so we need branching based on the running platform or just remove this line.